### DISCUSSION TITLE
[Fedora 39] The plugin does not work.

### DISCUSSION THREAD

**Discussion by ghost** [wojtryb/Shortcut-Composer#75] [Help me]
It does not import correctly, Krita returns no error, it just doesn't show up in the app after an import.

**Comment by wojtryb** [comment]
Fedora is not a supported distribution.

You probably haven't met the [requirements](https://github.com/wojtryb/Shortcut-Composer#requirements) - either you are not using appimage version of krita, or krita/plugin versions are too old.
